xapian-core/cluster/cluster.cc

```diff
@@ -52,14 +52,35 @@ Clusterer::~Clusterer()
     LOGCALL_DTOR(API, "Clusterer");
 }
 
+TermListGroup::TermListGroup()
+    : stopper(NULL)
+{
+    LOGCALL_CTOR(API, "TermListGroup", NO_ARGS);
+}
+
 TermListGroup::TermListGroup(const MSet &docs)
+    : stopper(NULL)
 {
     LOGCALL_CTOR(API, "TermListGroup", docs);
+    construct_tlg(docs);
+}
+
+void
+TermListGroup::construct_tlg(const MSet &docs)
+{
```

Comment:
I don't see construct_tlg being used at two places. Why a different function which needs to be called inside the constructor. Why can't we have code inside this function inside the actual constructor? Do we have any plans to use it in two constructors?